test/test_fs_after_main.c

```diff
@@ -60,10 +62,10 @@ int main() {
       counter++;
       if (counter < 5) {
         out('js queueing');
-        setTimeout(looper, 1);
+        safeSetTimeout(looper, 1);
       } else {
         out('js finishing');
-        setTimeout(Module['_finish'], 1);
+        safeSetTimeout(Module['_finish'], 1);
```

Comment:
Can you explain why this change is needed? A comment might be good in the code too.